test/test_transforms.py

```diff
@@ -305,5 +305,120 @@ def _test_librosa_consistency_helper(n_fft, hop_length, power, n_mels, n_mfcc, s
         _test_librosa_consistency_helper(**kwargs3)
 
 
+import pytest
+import numpy as np
+import torchaudio.functional as F
+xfail = pytest.mark.xfail
+
+
+@pytest.mark.parametrize('fft_length', [512])
+@pytest.mark.parametrize('hop_length', [256])
+@pytest.mark.parametrize('waveform', [
+    (torch.randn(1, 100000)),
+    (torch.randn(1, 2, 100000)),
+    pytest.param(torch.randn(1, 100), marks=xfail(raises=RuntimeError)),
+])
+@pytest.mark.parametrize('pad_mode', [
+    # 'constant',
+    'reflect',
+])
+def test_stft(waveform, fft_length, hop_length, pad_mode):
```

Comment:
stft as a pytorch operator should be properly tested.

Reply:
pytorch tests their own method so it would be interesting to see what we need to test on our end

https://github.com/pytorch/pytorch/blob/042da2171e95ae5cf39452ae17a5000a5a24eb85/test/test_torch.py#L6374-L6393